### DISCUSSION TITLE
What is the format for translating a Japanese student ID card?

### DISCUSSION THREAD

**Discussion by beli68** [community/community#52903] [GitHub Education]
I'm not very good at English and use online translators a lot, so I'm sorry if my sentences sound weird.
I have now reverted my profile to anonymous.

I am a Japanese student, and my application has been rejected many times.

I see on the forum that I am required to copy a set of translations from Japanese to English.
Is it necessary to have all items in English?

Also, do you expect me to process the image of the student ID card and put the translation on it? And do you expect me to use a webcam to capture a printout of them?

Or should I take a picture of my student ID card and the paper listing the translations?

I also believe the information in your possession regarding location information is incorrect.
However, since I do not know my location information you have obtained or the location information of the university you have, I need to figure out what is wrong.
I can get to the campus by train in a little over an hour. At least "does not appear to have a campus location in your country. " is incorrect.

<img width="568" alt="image" src="https://user-images.githubusercontent.com/1407621/232229207-046848cd-eb8d-4634-b6bc-ff353bb61cf7.png">

**Comment by Ukuer** [comment]
Hi, I am a Chinese student, and I have the same problem. 
I think it may recognize different languages.
I just upload Chinese document. I tried 10 times, and get the same reply:
 "You are unlikely to be verified until you have completed your [GitHub user profile] with your full name as it appears in your academic affiliation document plus a short bio. "
I don' t konw what it's mean.
